config-template.js

```diff
@@ -0,0 +1,79 @@
+/**
+ * Copy this file to config.js and fill it in with your own information.
+ */
+module.exports = {
```

Comment:
That's why I had additional comments above each config option. I assumed some of them were fairly obvious (database username/password), not explaining them very well, and put default values where appropriate. The Github ones do link to the Github page, but it could be explained better. Additionally, I did add some code in certain places to break better when certain configs are missing.

I suppose I don't know what and/or how to improve it much without feeling in the dark, or having someone totally new to it follow the steps, get confused, and ask some questions.

After writing all of this comment I feel I should just improve the readme (as you stated earlier and I forgot) and have you give feedback after that.